spec/spec_helper.rb

```diff
@@ -25,7 +25,7 @@ def mock_watir_browser
 
 def mock_selenium_browser
   selenium_browser = double('selenium')
-  allow(selenium_browser).to receive(:is_a?).with(Watir::Browser).and_return(false)
+  allow(selenium_browser).to receive(:is_a?).with(anything).and_return(false)
```

Comment:
Had to change this because now it will also receive watir elements and should return false.